src/setupTests.js

```diff
@@ -40,10 +42,12 @@ global.auth0 = {
   })),
 };
 global.localStorage = mockLocalStorage();
+global.sessionStorage = mockLocalStorage();
 global.mount = mount;
 global.shallow = shallow;
 global.React = React;
 global.ReactDOM = ReactDOM;
+global.Headers = Headers;
```

Comment:
If you don't do this you get unhandled errors in `api.js`. In Jest they appear as warnings like this:

```
(node:646) UnhandledPromiseRejectionWarning: ReferenceError: Headers is not defined
(node:646) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:646) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:646) UnhandledPromiseRejectionWarning: ReferenceError: Headers is not defined
(node:646) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 8)
```

You can hopefully see an example of that here: https://circleci.com/gh/mozilla/delivery-console/3515

It saddens me that the tests don't fail on this blatant error (that global `Headers` obviously doesn't exist in NodeJS).

Reply:
Also, when using globals, should one use `window.` instead? E.g. `window.localStorage` instead of `localStorage`? 
You know, to avoid this happening again.

Reply:
Or rather, to be more explicit.